config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -2833,7 +2833,7 @@
 
 // Set number of user-controlled fans. Disable to use all board-defined fans.
 // :[1,2,3,4,5,6,7,8]
-//#define NUM_M106_FANS 1
+#define NUM_M106_FANS 1
```

Comment:
Only one controllable fan on this machine.
If not defined then we have 2 fan in the menus